soh/soh/Enhancements/mods.cpp

```diff
@@ -258,14 +256,15 @@ void RegisterOcarinaTimeTravel() {
 
 void AutoSave(GetItemEntry itemEntry) {
     u8 item = itemEntry.itemId;
+    bool performSave = false;
     // Don't autosave immediately after buying items from shops to prevent getting them for free!
     // Don't autosave in the Chamber of Sages since resuming from that map breaks the game
     // Don't autosave during the Ganon fight when picking up the Master Sword
     // Don't autosave in the fishing pond to prevent getting rod on B outside of the pond
     // Don't autosave in the bombchu bowling alley to prevent having chus on B outside of the minigame
     // Don't autosave in grottos since resuming from grottos breaks the game.
```

Comment:
nit:  this comment should be updated to remove the situations we no longer don't save in

Reply:
True.